library/src/main/java/com/google/android/exoplayer2/ExoPlayerImplInternal.java

```diff
@@ -423,13 +435,9 @@ private void updatePlaybackPositions() throws ExoPlaybackException {
     if (periodPositionUs != C.TIME_UNSET) {
       resetRendererPosition(periodPositionUs);
     } else {
-      if (rendererMediaClockSource != null && !rendererMediaClockSource.isEnded()) {
-        rendererPositionUs = rendererMediaClock.getPositionUs();
-        standaloneMediaClock.setPositionUs(rendererPositionUs);
-      } else {
-        rendererPositionUs = standaloneMediaClock.getPositionUs();
-      }
-      periodPositionUs = playingPeriodHolder.toPeriodTime(rendererPositionUs);
+      rendererPositionUs = rendererMediaClock.getPositionUs();
```

Comment:
It will lead the movies with video only crash;

**_It is because trying to access rendererMediaClock but with a video-only movie, it is NULL;_**

Please fix it, thanks